### ISSUE TITLE
Readme does not help

### ISSUE THREAD

**Issue by mortalisk** [equinor/videx-wellog#232]
**Describe the bug**
Try to follow readme in a new setup

```
npm run docs

> @equinor/videx-wellog@0.9.1 predocs
> rimraf docs


> @equinor/videx-wellog@0.9.1 docs
> typedoc --out docs src --module es5

error Invalid structure in plugin typedoc-neo-theme, no load function found.
error Unknown option: --module
error Tried to set an option (mode) that was not declared.
error Tried to set an option (excludeNotExported) that was not declared.
```

Story book:
```
cd examples/plain-javascript-exampl
bash: cd: examples/plain-javascript-exampl: No such file or directory
```

**Expected behavior**
Some useful help

**Additional context**
Trying to see the capabilites of the code or how it is used in any way. Some example code for each possible visualisation, along with a screenshot, would probably be the most useful documentation.

**Comment by fbosquet-azpn** [comment]
Note regarding the storybook issue, it is addressed in the following pull request #228

**Comment by fbosquet-azpn** [comment]
Note regarding the type-neo-theme, 
This theme does no longer supported since version 0.22 of typedoc
https://github.com/google/typedoc-neo-theme/issues/75

Here is a list of additional theme supported by typedoc
https://typedoc.org/guides/themes/
